lib/node_modules/@stdlib/math/base/special/nonfibonacci/README.md

```diff
@@ -123,6 +123,91 @@ for ( i = 1; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/nonfibonacci.h"
+```
+
+#### stdlib_base_nonfibonacci( x )
+
+Computes the nth [non-Fibonacci number][fibonacci-number].
+
+```c
+double out = stdlib_base_nonfibonacci( 1 );
+// returns 4
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+
+```c
+double stdlib_base_nonfibonacci( const double x );
```

Comment:
We should follow `@stdlib/math/base/special/fibonacci` here: https://github.com/stdlib-js/stdlib/blob/31632017d48d98b52623397755fe880b08eb1cb4/lib/node_modules/%40stdlib/math/base/special/fibonacci/src/main.c#L114. Namely, `int32_t` in and `double` out.

Reply:
But here input for `nonfibonacci` can be greater than 2**32 - 1, no?

Reply:
In general when we wish to allow `NaN` we will prefer `double`? and same for output.

Reply:
We can make it `int64_t` then.

Re: `NaN`. I thought we already discussed that for `fibonacci`. There we don't support `NaN` inputs in C. Why would we do so here? IIRC, we require the C `fibonacci` to be `int32_t`, as unlikely that, in C, you'd want to provide anything other than an `int`. Not sure why this case would be any different.

Reply:
Here is discussion about fibonacci on why we opted for double as output https://github.com/stdlib-js/stdlib/pull/783#discussion_r1057933901

Reply:
Perfect, and same question for `gcd( a, b )` this will go as `stdlib_base_gcd( const int64_t a, const int64_t b )`

Reply:
Re: gcd. That seems fine with me.

Reply:
I am aligned on having `double` as output for this API.